testsupport/tiers/templaterefs.go

```diff
@@ -20,9 +21,15 @@ func GetTemplateRefs(hostAwait *wait.HostAwaitility, tierName string) TemplateRe
 	for _, ns := range templateTier.Spec.Namespaces {
 		nsRefs = append(nsRefs, ns.TemplateRef)
 	}
+	spaceRoleRefs := make(map[string]string, len(templateTier.Spec.SpaceRoles))
+	for role, ns := range templateTier.Spec.SpaceRoles {
+		spaceRoleRefs[role] = ns.TemplateRef
+	}
+
 	return TemplateRefs{
 		Namespaces:       nsRefs,
 		ClusterResources: clusterResourcesRevision(*templateTier),
+		SpaceRoles:       spaceRoleRefs,
```

Comment:
I have one more comment - where are the spaceRoleRefs verified?

Reply:
I believe this was addressed in b169796